applications/dashboard/controllers/class.settingscontroller.php

```diff
@@ -2049,7 +2049,7 @@ public function mobileThemes($ThemeName = '', $TransientKey = '') {
      * @param int|string $userIdentifier Either the username or user ID.
      */
     private function findBanRule($userIdentifier) {
-        $this->permission('Moderation.Bans.Manage');
+        $this->permission('Garden.Settings.Manage');
```

Comment:
This permission check seems redundant. `SettingsController::findBanRule` is a private method. The only place it's called is in `SettingsController::bans`, which [already has a permission check at the top of it](https://github.com/vanilla/vanilla/blob/cbb18ce9e5be5eb85126d723ba0fd2d3a98dacff/applications/dashboard/controllers/class.settingscontroller.php#L568).